packages/lodestar-params/src/interface.ts

```diff
@@ -70,4 +70,17 @@ export interface IBeaconParams {
   MAX_ATTESTATIONS: number;
   MAX_DEPOSITS: number;
   MAX_VOLUNTARY_EXITS: number;
+
```

Comment:
I define ALL_FORKS here and load it from yaml file in lodestar-cli, is it ok? I want it to be loaded in 1 single space and be able to process it in multiple modules in `config.params.ALL_FORKS`. Maybe for historical hard forks we want to store it in database, just load the future ones.

Reply:
This is fine